package.json

```diff
@@ -34,9 +30,9 @@
     "chai": "^4.1.2",
     "mocha": "^5.0.5",
     "nock": "^9.2.3",
-    "sinon": "^4.5.0",
+    "sinon": "^7.4.0",
     "ts-node": "^5.0.1",
-    "tslint": "^5.9.1",
+    "tslint": "^5.18.0",
```

Comment:
https://github.com/palantir/tslint/compare/5.9.1...5.18.0 similar situation to above ✅